digdag-tests/src/test/java/acceptance/MigrationIT.java

```diff
@@ -20,7 +20,14 @@
 
 public class MigrationIT
 {
-    public TemporaryDigdagServer server = null;
+    // Shouldn't setup TemporaryDigdagServer using @Rule in this test
+    // since the annotation implicitly calls `apply()` method, but this test
+    // expects the method hasn't been called
+    /*
+    @Rule
+    public TemporaryDigdagServer server = TemporaryDigdagServer.of();
+    */
+    private TemporaryDigdagServer server = null;
```

Comment:
I don't have strong opinion:
This code set `null` but PyIT you fixed does not set it.
I guess  it is better to be consistent.